book.bib

```diff
@@ -1397,3 +1397,49 @@ @Article{ZIBB2018
 }
 
 
+
+@article{yoon2019microbial,
+  title={Microbial networks in SPRING-Semi-parametric rank-based correlation and partial correlation estimation for quantitative microbiome data},
+  author={Yoon, Grace and Gaynanova, Irina and M{\"u}ller, Christian L},
+  journal={Frontiers in genetics},
+  volume={10},
+  pages={516},
+  year={2019},
+  publisher={Frontiers Media SA}
+}
+
+
+@article{peschel2021netcomi,
+    author = {Peschel, Stefanie and M{\"{u}}ller, Christian L and von Mutius, Erika and Boulesteix, Anne-Laure and Depner, Martin},
+    title = "{NetCoMi: network construction and comparison for microbiome data in R}",
```

Comment:
Same, check double curly brackets since capitalization carries meaning here. 

The quotes are unnecessary I think?